web/app/js/components/ConfigureProfilesMsg.jsx

```diff
@@ -29,21 +32,30 @@ const styles = theme => ({
   textField: {
     marginLeft: theme.spacing.unit,
     marginRight: theme.spacing.unit,
+    marginTop: 2 * theme.spacing.unit,
     width: 200,
   },
   root: {
     marginTop: theme.spacing.unit * 3,
     marginBottom: theme.spacing.unit * 3,
   },
 });
+
+const dns1123ServiceFmt = '^[a-z]([-a-z0-9]*[a-z0-9])?$';
```

Comment:
this should be `dns1035ServiceFmt ` as we're using the dns1035 check for service and the dns1123 check for namespace, so it'd be confusing if they both had this in the variable name.